### PR TITLE
Add descriptions to ListView

### PR DESCRIPTION
**Pull request by reidbarber**
Separating from #2734

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 🧢 Your Project:

RSP

### CODE REVIEW

**Review by dannify — APPROVED** [review]

**Comment by dannify** [comment]
I want to change the description for Listview to something else but this is fine for now.

**Review by ktabors — APPROVED** [review]

File: packages/@react-spectrum/list/src/ListView.tsx [hunk 1]
```diff
@@ -88,11 +88,26 @@ interface ListViewProps<T> extends CollectionBase<T>, DOMProps, AriaLabelingProp
...
+   */
   transitionDuration?: number,
+  /**
+   * Handler that is called when a user performs an action on an item. The exact user event depends on
+   * the collection's `selectionBehavior` prop and the interaction modality.
+   */
   onAction?: (key: string) => void,
+  /**
+   * The drag hooks returned by `useDragHooks` used to enable drag and drop behavior for the ListView. See the
+   * [docs](https://react-spectrum.adobe.com/react-spectrum/useDragHooks.html) for more info.
```
Comment (LFDanLu) [inline packages/@react-spectrum/list/src/ListView.tsx:109]: note that this doc link doesn't exist yet but it should when the docs for ListView and stuff go out

File: packages/@react-spectrum/list/src/ListView.tsx [hunk 2]
```diff
@@ -88,11 +88,26 @@ interface ListViewProps<T> extends CollectionBase<T>, DOMProps, AriaLabelingProp
...
   isQuiet?: boolean,
+  /** The current loading state of the ListView. Determines whether or not the progress circle should be shown. */
   loadingState?: LoadingState,
+  /** Sets what the ListView should render when there is no content to display. */
   renderEmptyState?: () => JSX.Element,
+  /**
+   * The duration of animated layout changes, in milliseconds. Used by the Virtualizer.
+   * @default 0
+   */
   transitionDuration?: number,
```
Comment (LFDanLu) [inline packages/@react-spectrum/list/src/ListView.tsx:101]: As an aside, I don't think this should be in the ListView api, not sure why it is here in the first place. Probably should  get removed and match TableView, will do in a separate PR

**Review by LFDanLu — APPROVED** [review]

File: packages/@react-spectrum/list/src/ListView.tsx (hunk 2 above)
Comment (reidbarber) [reply packages/@react-spectrum/list/src/ListView.tsx:101]: Definitely agree

File: packages/@react-stately/calendar/package.json [hunk 3]
```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "@babel/runtime": "^7.6.2",
     "@internationalized/date": "3.0.0-alpha.4",
+    "@react-aria/i18n": "^3.3.8",
```
Comment (snowystinger) [inline packages/@react-stately/calendar/package.json]: seems erroneous

Comment (LFDanLu) [reply packages/@react-stately/calendar/package.json]: yup, think I had some accidental left over changes in my editor. Removed by https://github.com/adobe/react-spectrum/pull/2850/commits/b319c25dd80b5e150511b4a6e4e894edf684e96e

**Review by snowystinger — APPROVED** [review]
